src/test/java/seedu/address/ui/NearbyPersonListPanelTest.java

```diff
@@ -55,4 +56,15 @@ public void display() {
         assertCardDisplaysPerson(expectedPerson, actualCard);
         assertEquals(Integer.toString(1) + ". ", actualCard.getId());
     }
+
+    @Test
+    public void handleJumpToNearbyListRequestEvent() {
```

Comment:
same issue as above about the naming of function

Reply:
this one seems okay to me though